ts/session/messages/outgoing/content/ContentMessage.ts

```diff
@@ -0,0 +1,28 @@
+import { Message } from '../Message';
+import { SignalService } from '../../../../protobuf';
+
+export abstract class ContentMessage implements Message {
+  public readonly timestamp: number;
+  public readonly identifier: string;
+
+  constructor(timestamp: number, identifier: string) {
+    this.timestamp = timestamp;
+    this.identifier = identifier;
```

Comment:
What are your thoughts on just setting identifier to a uuid and then having a public function `setIdentifier(identifer: string)` which can be called if a custom identifier is needed?

Custom identifiers should only be needed when we need to somehow keep track of the message, and there are only specific instances in which we would want to do that.

Reply:
Since we are using dictionaries now, might be better instead to have `identifier` be a string or undefined and thus we can do the above without having the function. Any method will do

Reply:
agreed